writing/alcohol-drugs.md

```diff
@@ -0,0 +1,26 @@
+# Balancing Alcohol, Drugs and Fun
+
+There's no way around it, alcohol and drugs are super fun. Anything that can help you relax, have fun or open your mind sounds great in my books. I've developed some of my best (and often unusual) ideas when slightly drunk or high. And the impact of those ideas lasts way past the effects of the substance.
```

Comment:
"sounds great in my book" doesn't really work... "sounds great to me" or "is great in my book"?